### ISSUE TITLE
UX: Timeline mouse cursor 

### ISSUE THREAD

**Issue by guettli** [jlfwong/speedscope#241]
This is related to #240. An easier way to understand that you can set the viewport in the upper part.

Up to now the mouse cursor is like in this screenshot:

![image](https://user-images.githubusercontent.com/414336/69531130-a01afb80-0f73-11ea-9b8b-962019943704.png)

It would be more intuitive if the cursor would look like this: https://www.iconfinder.com/icons/927852/cursor_design_development_drag_pointer_select_icon

**Comment by jlfwong** [comment]
Hi @guettli, thanks for the suggestion!

At present, all of the cursors used in speedscope are default browser cursors. I haven't spent any time investigating custom cursors for speedscope, but I agree this icon isn't particularly good at indicating what the interaction is going to do.

**Comment by guettli** [comment]
Here are the css cursor styles which are available by default: https://developer.mozilla.org/de/docs/Web/CSS/cursor

I think "crosshair" would fit: From chrome:
![image](https://user-images.githubusercontent.com/414336/69630594-5ef71a00-104d-11ea-94c6-ac225fb83285.png)

**Comment by jlfwong** [comment]
Ah, I originally didn't choose the crosshair one because the I thought the crosshair would suggest that the range being dragged was two dimensional. the minimap dragging only actually sets the range on a single axis though. What do you think? Maybe the crosshair is still clearer.

**Comment by guettli** [comment]
Yes, I think a crosshair is more intuitive. I hardly never seen the two headed arrow symbol.

**Comment by jlfwong** [comment]
@guettli I'd accept a PR to change this cursor if you'd like to do it! The relevant code is here: https://github.com/jlfwong/speedscope/blob/bdd9301c59f29c105762b6460d731438c670d705/src/views/flamechart-minimap-view.tsx#L363-L375

**Comment by guettli** [comment]
Sorry, my JS knowledge is limited and I have no developer to delegate this at the moment.
